bucket/fiddler.json

```diff
@@ -5,17 +5,31 @@
     "license": "Unknown",
     "url": "https://telerik-fiddler.s3.amazonaws.com/fiddler/FiddlerSetup.exe#/dl.7z",
     "hash": "aaa4d714eeaae8f6a96a63110cd239ec44fac75ae591ab181c13ac837e0ec1a4",
-    "installer": {
-        "script": "Expand-7zipArchive \"$dir\\`$PLUGINSDIR\\FiddlerSetup.exe\" \"$dir\" -Removal"
+    "pre_install": [
+        "Expand-7zipArchive \"$dir\\`$PLUGINSDIR\\FiddlerSetup.exe\" \"$dir\" -Removal",
+        "Remove-Item \"$dir\\`$*\" -Recurse",
+        "    # Script editor",
+        "Move-Item \"$dir\\FSE2.exe\" \"$dir\\ScriptEditor\"",
+        "$path = 'HKCU:\\Software\\Microsoft\\Fiddler2\\InstallerSettings'",
+        "New-Item $path -ErrorAction SilentlyContinue -Force | Out-Null",
+        "@(@('InstallPath', ''), @('PluginPath', 'Inspectors'), @('ScriptPath', 'Scripts')) | ForEach-Object {",
+        "    New-ItemProperty -Path $path -Name $_[0] -Value \"$dir\\$($_[1])\" -PropertyType ExpandString -ErrorAction SilentlyContinue | Out-Null",
+        "}"
+    ],
+    "uninstaller": {
+        "script": "If ($purge) { Remove-Item 'HKCU:\\Software\\Microsoft\\Fiddler2' -Force -Recurse }"
     },
-    "post_install": "Remove-Item \"$dir\\`$PLUGINSDIR\" -Recurse",
-    "bin": "Fiddler.exe",
```

Comment:
Is there a reason for removing bin?

Reply:
It's been my mistake, thanks for pointing it out. I'll bring the `bin` back...